services/api/Dockerfile.jobs

```diff
@@ -6,10 +6,13 @@ COPY package.json /service/package.json
 COPY yarn.lock /service/yarn.lock
 
 # Update & install required packages
-RUN apk add --update bash curl;
+RUN apk add --update bash jq curl gcc musl-dev python3 python3-dev;
+RUN python3 -m pip install --user pipx yq
+RUN python3 -m pipx ensurepath
+RUN python3 -m pipx install yacron
+RUN wget https://github.com/mikefarah/yq/releases/download/v4.6.3/yq_linux_386.tar.gz -O - | tar xz && mv yq_linux_386 /usr/bin/yq
```

Comment:
wondering if this is best practices in a docker file (don't know myself) cc @beatlevic

Reply:
Normally I would use `apk` but this tool is too exotic for that. This tool is needed to deal with YAML